### PR TITLE
CLOUDP-161161: [Atlas CLI] Add support for --curentIp on API access list for organizations

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-161161

## Checklist
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

Manually tested, adding e2e has a limitation as the backend won't let you remove the callers IP address

### CODE REVIEW

File: docs/atlascli/command/atlas-organizations-apiKeys-accessLists-delete.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ Syntax
 
 .. code-block::
 
-   atlas organizations apiKeys accessLists delete <ID> [options]
+   atlas organizations apiKeys accessLists delete <entry> [options]
```
Comment (gssbzn) [inline docs/atlascli/command/atlas-organizations-apiKeys-accessLists-delete.txt:22]: renaming to entry as ID is not correct, we delete IPs or CIDR values

File: test/e2e/iam/org_api_key_access_list_test.go [hunk 2]
```diff
@@ -92,21 +88,54 @@ func TestOrgAPIKeyAccessList(t *testing.T) {
...
+			var key mongodbatlas.AccessListAPIKeys
+			if err := json.Unmarshal(resp, &key); a.NoError(err) {
+				a.NotEmpty(key.Results)
+				entry = key.Results[0].IPAddress
+			}
 		}
 	})
+
+	t.Run("Delete", func(t *testing.T) {
+		t.Skip("400 (request \"CANNOT_REMOVE_CALLER_FROM_ACCESS_LIST\") Cannot remove caller's IP address from access list")
```
Comment (gssbzn) [inline test/e2e/iam/org_api_key_access_list_test.go:118]: 🤷

File: docs/mongocli/command/mongocli-iam-organizations-apiKeys-accessLists-create.txt [hunk 3]
```diff
@@ -44,6 +44,10 @@ Options
      - strings
      - false
      - Access list entry in CIDR notation to be added for your API key. To add more than one entry, you can specify each entry with a separate cidr flag or specify all the entries as a comma-separated list using one cidr flag. You can't set both cidr and ip in the same command.
+   * - --currentIp
+     -
```
Comment (wtrocki) [inline docs/mongocli/command/mongocli-iam-organizations-apiKeys-accessLists-create.txt:48]: empty line ?

Comment (gssbzn) [reply docs/mongocli/command/mongocli-iam-organizations-apiKeys-accessLists-create.txt:48]: yes, this renders as a table, this is a column, it'll have no value, if you want to know more these docs are auto-generated from the code using https://github.com/mongodb-labs/cobra2snooty

**Review by wtrocki — APPROVED** [review]

**Review by fmenezes — APPROVED** [review]
LGTM

**Review by sarahsimpers — APPROVED** [review]
Sorry for delay, LGTM

**Pull request merged**